Documentation/CRDs/Cluster/ceph-cluster-crd.md

```diff
@@ -222,7 +222,7 @@ Based on the configuration, the operator will do the following:
           public: rook-ceph/rook-public-nw
     ```
 
-2. If only the `cluster` selector is specified, the internal cluster traffic* will happen on that network. All other traffic to mons, OSDs, and other daemons will be on the default network.
+2. If only the `cluster` selector is specified, the internal cluster traffic\* will happen on that network. All other traffic to mons, OSDs, and other daemons will be on the default network.
```

Comment:
The rendering of this looks fine [here](https://rook.io/docs/rook/latest/CRDs/Cluster/ceph-cluster-crd/#multus) without the escaping. Where did you say it's not showing up correctly?

Reply:
Not all markdown renderers are the same underlying implementation. This is the correct syntax to use if one desires a literal asterisk character, and the change will ensure it renders correctly with any rendering changes we or Github might make.